<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="qTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>62</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashLine</enum>
   </property>
   <property name="rowCount">
    <number>2</number>
   </property>
   <property name="columnCount">
    <number>10</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>60</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>60</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <row>
    <property name="text">
     <string>Ki</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Yi</string>
    </property>
   </row>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="qPushButtonRandom">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>СЛУЧАЙНЫЕ
ЧИСЛА</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qPushButtonClear">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>ОЧИСТИТЬ
ТАБЛИЦУ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qPushButtonSolver">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>ВЫПОЛНИТЬ
ЗАДАНИЕ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qPushButtonExit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>ВЫХОД</string>
   </property>
  </widget>
  <widget class="QLabel" name="qLabelPicturePrimer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>620</width>
     <height>163</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
